src/utils/getUserIdentityFromId.ts

```diff
@@ -0,0 +1,23 @@
+import { IncomingMessage } from "http";
+import { GetServerSidePropsContext } from "next";
+
+type ServerRouter = {
+  route: string;
+  pathname: string;
+  query: Record<string, unknown>;
+  asPath: string;
+  basePath: string;
+  events: any;
+  isFallback: boolean;
+};
+
+export async function getUserIdentityFromId(
+  context: GetServerSidePropsContext,
+): Promise<void> {
+  const serverRouter: ServerRouter = (context.req as IncomingMessage & {
+    ServerRouter: ServerRouter;
+  }).ServerRouter;
+  console.log(serverRouter);
+
+  serverRouter && console.log(serverRouter.query.id);
+}
```

Comment:
@Fenntasy I left this file to show you how I'll query management when we'll be able to filter identities with the ID, and not the value (you can refer to the conv in slack's `connect` channel).

Two things worries me:
- `ServerRoute` is not type out of the box, which worries me.
- When I log the object, it first logs as `undefined`, then as the actual object. So the correct type would be:
```ts
ServerRouter = (context.req as IncomingMessage & {
    ServerRouter?: ServerRouter;
  }
```

This is all the only implementation I can think of, and I'm really not a fan of this. I pinged you on the slack conv to see if I did not miss anything.

Reply:
I'm sorry, I don't understand what you mean there.

This file would be called by the future file `/account/logins/${email.type}/${email.id}`? 
I don't understand what is `serverRouter` to be honest